js/main.js

```diff
@@ -1,2 +1,44 @@
 /* global data */
 /* exported data */
+var $title = document.querySelector('#note-title');
+var $urlInput = document.querySelector('#url');
+var $notes = document.querySelector('#entry-notes');
+var $submit = document.querySelector('form');
+
+function setImage(event) {
+
+  document.querySelector('img').setAttribute('src', $urlInput.value);
+}
+
+$urlInput.addEventListener('blur', setImage);
+
+function resetForm(event) {
+  data.entries.push({
+    title: $title.value,
+    photoURL: $urlInput.value,
+    notes: $notes.value
+  });
+  data.nextEntryId += 1;
```

Comment:
You did not add a `entryId` property to the new entry before incrementing this, please do so